commands/make_producer.ts

```diff
@@ -0,0 +1,69 @@
+import { BaseCommand, args, flags } from '@adonisjs/core/ace'
+import string from '@adonisjs/core/helpers/string'
+import { VariableDeclarationKind } from 'ts-morph'
+import { stubsRoot } from '../stubs/main.js'
+import { CommandOptions } from '@adonisjs/core/types/ace'
+
+export default class MakeProducer extends BaseCommand {
```

Comment:
This file seems to not be emitted using the previous tsc / tsup configuration, hence needing to change.